src/harness/certs/generate_fake_certs.sh

```diff
@@ -123,10 +123,6 @@ openssl ca -cert sas_ca.cert -keyfile private/sas_ca.key -in admin_client.csr \
     -policy policy_anything -extensions cbsd_req_sign -config ../../../cert/openssl.cnf \
     -batch -notext -create_serial -utf8 -days 1185 -md sha384
 
-
```

Comment:
Please remove unrelated changes from this PR.

Reply:
Removed unrelated   changes
--